client/Layout/SelectedCombatants.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from "react";
+import { useEffect } from "react";
 import { useSubscription } from "../Combatant/linkComponentToObservables";
 import { CombatantDetails } from "../Combatant/CombatantDetails";
 import { Button } from "../Components/Button";
 import { CombatantCommander } from "../Commands/CombatantCommander";
 import { MultipleCombatantDetails } from "../Combatant/MultipleCombatantDetails";
+import { NotifyTutorialOfAction } from "../Tutorial/NotifyTutorialOfAction";
 
-export function SelectedCombatants(props: {
-  combatantCommander: CombatantCommander;
-}) {
+export function SelectedCombatants(props: { combatantCommander: CombatantCommander }) {
   const { combatantCommander } = props;
-  const selectedCombatants = useSubscription(
-    combatantCommander.SelectedCombatants
-  );
+  const selectedCombatants = useSubscription(combatantCommander.SelectedCombatants);
+
+  useEffect(() => {
+    if (selectedCombatants.length > 0) {
+      NotifyTutorialOfAction("combatant-click");
```

Comment:
This should be switched to a CamelCased string, maybe something like "CombatantSelected"